app/scripts/migrations/139.ts

```diff
@@ -0,0 +1,79 @@
+import { hasProperty, isObject } from '@metamask/utils';
+import { cloneDeep } from 'lodash';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 138;
+
+/**
+ * This migration deletes properties from state which have been removed in
+ * previous commits.
+ *
+ * @param originalVersionedData - Versioned MetaMask extension state, exactly
+ * what we persist to dist.
+ * @param originalVersionedData.meta - State metadata.
+ * @param originalVersionedData.meta.version - The current state version.
+ * @param originalVersionedData.data - The persisted MetaMask state, keyed by
+ * controller.
+ * @returns Updated versioned MetaMask extension state.
+ */
+export async function migrate(
+  originalVersionedData: VersionedData,
+): Promise<VersionedData> {
+  const versionedData = cloneDeep(originalVersionedData);
+  versionedData.meta.version = version;
+  transformState(versionedData.data);
+  return versionedData;
+}
+
+function transformState(state: Record<string, unknown>) {
+  if (
+    hasProperty(state, 'AppStateController') &&
+    isObject(state.AppStateController)
+  ) {
+    // Removed in 4ea52511eb7934bf0ce6b9b7d570a525120229ce
+    delete state.AppStateController.serviceWorkerLastActiveTime;
+  }
+
+  if (
+    hasProperty(state, 'NetworkController') &&
+    isObject(state.NetworkController)
+  ) {
+    // Removed in 555d42b9ead0f4919356ff16e11c663c5e38639e
+    delete state.NetworkController.networkConfigurations;
+    // Removed in 800a9d3a177446ff2d05e3e95ec06b3658474207 with a migration, but
+    // still persists for some people for some reason
+    delete state.NetworkController.providerConfig;
```

Comment:
I found this error in Sentry: https://metamask.sentry.io/issues/6011869130/events/039861ddb07f4b39b947edba3bbd710e/. I am not sure why it is occurring. We already have a migration that removes this property. So do we need this here? Kind of strange that some people still have this property.

Reply:
It must be due to some migration failure that we have not yet understood. I think it is good to attempt to delete again here, but we should investigate further if it continues to persist after this